community/openwhisk/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+apiVersion: v1
+description: An open source, distributed serverless platform that executes functions in response to events at any scale
+name: openwhisk
+version: 0.1.0
+icon: https://raw.githubusercontent.com/apache/incubator-openwhisk/682eb5b62ee6ba8017ab54226c2ace3637f4f1ec/docs/images/whisk_icon_full-color_with_tm_64x64-300dpi.png
+keywords:
+  - Apache OpenWhisk
+  - amd64
+  - ICP
+maintainers:
+  - name: Apache OpenWhisk committers
+    email: dev@openwhisk.apache.org
+tillerVersion: ">=2.9.0"
+kubeVersion: "1.10 - 1.11.*"
```

Comment:
Will this chart not work on kube version 1.12 and beyond?  Just curious why this isn't:
`kubeVersion: ">=1.10.0"`

Reply:
I haven’t tried it.  I don’t think there will be major problems, but hadn’t gotten my hands on a 1.12 cluster yet.

Reply:
set to >=1.10.0